### ISSUE TITLE
Publish App on F-Droid

### ISSUE THREAD

**Issue by Logianer** [Schrankian/campus-dual-app#18]
Hey,
is there a chance this app will be published on the FOSS-Appstore 'F-Droid'? Even in its beta-state?

**Comment by Schrankian** [comment]
Yeah i think it meets the requirements. If I find the time, I will try to adjust the Pipeline to also publish to F-Droid.

**Comment by Schrankian** [comment]
I read a bit throught the F-Droid documentation and found that it is a little bit more complicated than i initially thought. Since i currently have littel free time and it is already possible to just download the apk from this repo, I probably won't do this anytime soon. 
But if you feel comfortable, feel free to implement it yourself and open a pull request. If you want, I can give you the metadata i used for the Play Store.
